src/ui/pages/Menu/Menu.tsx

```diff
@@ -102,50 +114,52 @@ const Menu = () => {
         <IonGrid>
           <IonRow className="menu-input-row">
             <MenuItem
-              index={1}
+              index={SubMenuKey.Profile}
```

Comment:
Looks good except `index` is a `number` in the props of `MenuItem`. So maybe update that prop.

```
          onClick={() => showSelectedOption(index)}
          data-testid={`menu-input-item-${index}`}
```

And make sure that the `data-testid` is auto casting to an integer here.